tools/cli/go-whisk-cli/commands/api.go

```diff
@@ -27,6 +29,7 @@ import (
     "../wski18n"
 
     "github.com/fatih/color"
+    "github.com/ghodss/yaml"
```

Comment:
Ignore my comments,
It looks like ghodss/yaml depends on https://github.com/ghodss/yaml/blob/master/yaml.go#L10

is smaller library to make life easier to deal with json->yaml